CHANGELOG.md

```diff
@@ -2029,3 +2029,34 @@ This is a full list of proofs that have changed form to use irrelevant instance
   Inverse⇒Injection : Inverse S T → Injection S T
   ↔⇒↣ : A ↔ B → A ↣ B
   ```
+
+* Added new isomorphisms to `Data/Fin/Properties`:
+  ```agda
+  2↔Bool : Fin 2 ↔ Bool
+  Fin[m^n]↔Fin[m]^n : ∀ m n → Fin (m ^ n) ↔ Fin m Vec.^ n
+  ```
+
+* Added new isomorphisms to `Data/Unit/Polymorphic/Properties`:
+  ```agda
+  ⊤↔⊤* : ⊤ {ℓ} ↔ ⊤*
+  ```
+
+* Added new isomorphisms to `Data/Vec/N-ary`:
+  ```agda
+  Vec↔N-ary : ∀ n → (Vec A n → B) ↔ N-ary n A B
+  ```
+
+* Added new isomorphisms to `Data/Vec/Recursive`:
+  ```agda
+  lift↔ : {A B : Set a} → ∀ n → A ↔ B → A ^ n ↔ B ^ n
+  ```
+
+* Added new functions to `Function/Properties/Inverse`:
+  ```agda
+  ↔refl ↔sym ↔trans
```

Comment:
Please can you update these names and add their types!

Reply:
I updated the definitions in CHANGELOG.